Cargo.lock

```diff
@@ -7348,7 +7238,7 @@ dependencies = [
  "forc-client",
  "forc-pkg",
  "forc-test",
- "forc-tracing 0.63.6",
```

Comment:
probably best to do a search and replace to change all occurrences of 0.63.5 back to 0.63.6